synapseutils/monitor.py

```diff
@@ -20,30 +20,29 @@ def notifyMe(syn, messageSubject='', retries=0):
     :param retries: Number of retries to attempt on failure (default=0)
 
     Example::
```

Comment:
The generated html is not showing a code block here. Need to add a newline after `Example::`